src/component/form-field.js

```diff
@@ -96,4 +97,10 @@ export class FormField {
     return type;
   }
 
+  elementChanged(element) {
+    this.element.uid = generatedUid(element.type);
```

Comment:
Add a `static elementCount = 0;`

And then replace with

```js
this.element.id = this.element.id || `sx-form-${element.type}-${element.key}${++FormField.elementCount}`;
```

And then move to `attached()`